core/services/pipeline/runner.go

```diff
@@ -267,7 +267,7 @@ func (r *runner) panickedRunResults(spec Spec) ([]TaskRunResult, error) {
 }
 
 func (r *runner) executeRun(ctx context.Context, txdb *gorm.DB, spec Spec, meta JSONSerializable, l logger.Logger) (TaskRunResults, bool, error) {
```

Comment:
Good catch on FM - I will add the jobID/name to the spec there. I did it this way because I'm imagining a follow up where we actually merge the pipeline_spec table into part of the jobs table to simplify. Not sure I follow the "and executeTaskRun" part - executeTaskRun is only called in the context of ExecuteRun ?

Reply:
Sorry I misunderstood how `executeTaskRun` was used. Ignore that part then